marc_to_solr/lib/traject_config.rb

```diff
@@ -840,20 +840,27 @@
   accumulator.replace(subjects)
 end
 
-# Adds lc and siku subject unstem_search fields
+# Homosaurus controlled vocabulary
```

Comment:
@maxkadel great! can you include in the comment the link to the specific vocabulary? https://homosaurus.org/

Reply:
Done!